packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -271,10 +285,39 @@ export class OfficeAddinUsageData {
     delete this.usageDataClient.context.tags["ai.user.accountId"]; // subscription
   }
 
+   /**
+   * Transform the project name by adddin '-test' suffix to it if necessary
+   */
+  private getEventName() {
```

Comment:
Could you please move the functions so they are in alphabetic order?

Reply:
I didn't even know this convention existed. 
Looking into the code, looks like the file doesn't follow this at all (I could change around the functions to fix that).
Until then, I've put it near the get function, which makes more sense.